core/math/geometry.cpp

```diff
@@ -1158,7 +1158,7 @@ Vector<Vector<Point2> > Geometry::_polypath_offset(const Vector<Point2> &p_polyp
 		case END_SQUARE: et = etOpenSquare; break;
 		case END_ROUND: et = etOpenRound; break;
 	}
-	ClipperOffset co;
+	ClipperOffset co(2.0, 0.25 * SCALE_FACTOR); // Defaults from ClipperOffset.
```

Comment:
If those are the default values, isn't it the same as relying on `ClipperOffset`'s default constructor?

Reply:
Clipper doesn't know about the fact that points are scaled up, so those defaults need to be scaled too. The first one is the `miter_limit` which is specified in multiples of `p_delta` offset and doesn't need to be scaled, so the default value just needs to be redefined here because what we're really interested in is "underscaled" `arc_tolerance` which operates in units/pixels.

Clipper 10.0.0 will have two separate classes for dealing with integer and float coordinates (Clipper and ClipperD) so these kind of workarounds won't need to be made.